pvr.vuplus/addon.xml.in

```diff
@@ -175,6 +175,18 @@
     <disclaimer lang="zh_TW">這是測試版軟體！其原創作者並無法對於以下情況負責，包含：錄影失敗，不正確的定時設定，多餘時數，或任何產生的其它不良影響...</disclaimer>
     <platform>@PLATFORM@</platform>
     <news>
+v4.6.0
```

Comment:
This should read v4.7.0, actual changelog for v4.6.0 is missing and more important, you forgot to increase the addon version number in line 4 of this file.

Reply:
Ya, I did this in iptvsimple too since the API bump :(